src/lib/TranscriptEditor/MediaPlayer/index.module.css

```diff
@@ -28,11 +22,13 @@ video {
 }
 
 .title {
-  margin: 1em;
+  margin-left: 1em;
   color: white;
   height: 1.3em;
+  width: 97vw;
```

Comment:
We're going from quite nice round numbers to awkward pixel values and numbers like 98%, 97vw, `-2px`, `width: 97px !important`.

Just in general, might be worth rethinking the layouts and padding, margins etc. rather than dealing with fiddly numbers

Reply:
Yeah, I feel like this can be revisited after adding the media query breakpoint for the responsive design.